lib/dune_file.ml

```diff
@@ -169,8 +169,12 @@ module Packages = struct
         | None -> { changed = false; stanzas = original; renames })
     | otherwise -> { changed = false; stanzas = otherwise; renames }
 
+  let is_tuareg = function
```

Comment:
it might be clearer if this function took a list of sexps rather than a single sexp. Tuareg-ness is a property of a dune file (which here is represented as a list of sexps) - not a property of an individual sexp, as it would be meaningless to call this function on (say) the second sexp in the file. I think making this change will also remove the need for explicit pattern matching in the places where this function is used.